sql/src/main/java/io/crate/operation/reference/sys/cluster/ClusterSettingsExpression.java

```diff
@@ -125,6 +124,32 @@ private void applySettings(List<Setting> clusterSettings, Settings newSettings)
                     values.put(name, newValue);
                 }
             }
+            applyNewSettingIfDeprecated(CrateSettings.INDICES_FIELDDATA_BREAKER_LIMIT,
```

Comment:
this should not be needed at all, old and new settings will update the same internal value anyway, so just returning the new setting should be enough